src/components/PoolOverview.tsx

```diff
@@ -75,141 +87,191 @@ export default function PoolOverview({
   }
   const hasShare = !!userShareData?.usdBalance.gt("0")
   const isMetapool = isMetaPool(formattedData.name)
+  const history = useHistory()
+  const disableText = isOutdated || shouldMigrate || poolData.isPaused
+  const chipLabel = useMemo(() => {
+    if ((isOutdated || shouldMigrate) && poolData.isPaused) {
+      return (
+        <span>
+          OUTDATED <br />& PAUSED
+        </span>
+      )
+    } else if (isOutdated || shouldMigrate) {
+      return <span>OUTDATED</span>
+    } else if (poolData.isPaused) {
+      return <span>PAUSED</span>
+    } else {
+      return null
+    }
+  }, [isOutdated, shouldMigrate, poolData.isPaused])
 
   return (
-    <div
-      className={classNames("poolOverview", {
-        outdated: isOutdated || shouldMigrate,
-      })}
+    <Paper
+      sx={{
+        p: theme.spacing(2, 3),
+        borderColor:
+          isOutdated || shouldMigrate
+            ? theme.palette.secondary.main
+            : theme.palette.other.divider,
+      }}
+      data-testid="poolOverview"
     >
-      <div className="left">
-        <div className="titleAndTag">
-          {isMetapool ? (
-            <Tooltip title={<React.Fragment>{t("metapool")}</React.Fragment>}>
-              <h4 className="title underline">{formattedData.name}</h4>
-            </Tooltip>
-          ) : (
-            <h4 className="title">{formattedData.name}</h4>
-          )}
-          {(shouldMigrate || isOutdated) && (
-            <Chip
-              variant="filled"
-              size="small"
-              label="OUTDATED"
-              color="secondary"
-            />
-          )}
-          {poolData.isPaused && (
-            <Chip variant="filled" size="small" label="PAUSED" color="error" />
-          )}
-        </div>
-        {hasShare && (
-          <div className="balance">
-            <span>{t("balance")}: </span>
-            <span>{`$${formattedData.userBalanceUSD}`}</span>
-          </div>
-        )}
-        <div className="tokens">
-          <span style={{ marginRight: "8px" }}>[</span>
-          {formattedData.tokens.map(({ symbol, icon }) => (
-            <div className="token" key={symbol}>
-              <img alt="icon" className="tokenIcon" src={icon} />
-              <span>{symbol}</span>
-            </div>
-          ))}
-          <span style={{ marginLeft: "-8px" }}>]</span>
-        </div>
-      </div>
+      <Grid container alignItems="center" spacing={1}>
+        <Grid item xs={12} lg={3}>
+          <Box>
+            <Box
+              display="flex"
+              color={disableText ? theme.palette.text.disabled : undefined}
+            >
+              <Tooltip title={isMetapool ? <div>{t("metapool")}</div> : ""}>
```

Comment:
FYI, if title of `Tooltip` is empty("") then tooltip don't appear.
https://mui.com/api/tooltip/#props